caasp-configmap-reload-image/caasp-configmap-reload-image.kiwi

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- OBS-AddTag: caasp/v4/configmap-reload:%%PKG_VERSION%%-rev<VERSION> caasp/v4/configmap-reload:%%PKG_VERSION%%-rev<VERSION>-build<RELEASE> -->
+
+<image schemaversion="6.9" name="caasp-configmap-reload-image" xmlns:suse_label_helper="com.suse.label_helper">
+  <description type="system">
+    <author>SUSE Containers Team</author>
+    <contact>containers@suse.com</contact>
+    <specification>Prometheus running on a SLES15 SP1 container guest</specification>
+  </description>
+  <preferences>
+    <type
+      image="docker"
+      derived_from="obsrepositories:/suse/sle15#15.1">
+      <containerconfig
+        name="caasp/v4/configmap-reload"
+        tag="%%PKG_VERSION%%"
+        maintainer="SUSE Containers Team &lt;containers@suse.com&gt;">
+        <entrypoint execute="/usr/bin/configmap-reload"/>
+        <subcommand clear="true"/>
+        <labels>
+          <suse_label_helper:add_prefix prefix="com.suse.caasp.v4">
+            <label name="org.opencontainers.image.description" value="Prometheus running on a SLES15 SP1 container guest"/>
```

Comment:
I guess this descriptions should be updated, looks like a copy & paste leftover from Prometheus image